horreum-java-client/pom.xml

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>horreum</artifactId>
+        <groupId>io.hyperfoil.tools</groupId>
+        <version>0.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>horreum-java-client</artifactId>
+
+    <url>https://horreum.hyperfoil.io/</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+
+        <resteasy-client.version>3.13.2.Final</resteasy-client.version>
+        <keycloack-admin-client.version>14.0.0</keycloack-admin-client.version>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>io.hyperfoil.tools</groupId>
+            <artifactId>horreum-api</artifactId>
+            <version>0.1-SNAPSHOT</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-hibernate-orm-panache-common</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-hibernate-orm</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-arc</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-core</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>io.hyperfoil.tools</groupId>
```

Comment:
This could be removed if I think about the deps more.  In the parent pom we exclude the graal sdk, and therefore the graalsdk is not pulled in as a dependency; however the client will not compile without the graal sdk on the classpath. 

I am thinking that we probably should not expose the Json class from yaup in the horreum api, due to the number of dependencies that it pulls into the project.  Maybe we should look at replacing Json in the api with a more lightweight class and using yaup Json internally if we need to?  

Also, I need to create a custom serializer for the yaup Json class, that we could possibly remove if we choose a different class to represent the JSON payload

Reply:
https://github.com/Hyperfoil/Horreum/issues/80

I would not block this PR though unless using YAUP requires further development. Let's integrate it with the 'garbage' dependencies and then get rid of YAUP in one shot.